src/targets/mambaf405us_i2c/target.h

```diff
@@ -0,0 +1,62 @@
+#include "config.h"
+
+// PORTS
+#define SPI_PORTS \
+  SPI1_PA5PA6PA7 \
+  SPI2_PB13PB14PB15 \
+  SPI3_PC10PC11PB5 \
+
+#define USART_PORTS \
+	USART1_PB7PB6 \
+	USART2_PA3PA2 \
+	USART3_PB11PB10 \
+	USART4_PA1PA0 \
+	USART5_PD2PC12 \
+	USART6_PC7PC6
+
+// RADIO
+#define RX_USART USART_PORT1
```

Comment:
we have stopped specifying these in production targets (they are still useful for dev tho)